packages/webdriverio/webdriverio.d.ts

```diff
@@ -3,21 +3,36 @@
 type BrowserObject = WebDriver.ClientOptions & WebDriver.ClientAsync & WebdriverIOAsync.Browser;
 
 // Element commands that should be wrapper with Promise
-type ElementPromise = Omit<WebdriverIO.Element, 'addCommand'>;
+type ElementPromise = Omit<WebdriverIO.Element, 'addCommand' | '$' | '$$'>;
 
 // Element commands wrapper with Promise
 type ElementAsync = {
     [K in keyof ElementPromise]: WrapWithPromise<ElementPromise[K], ReturnType<ElementPromise[K]>>
+} & {
```

Comment:
could we somehow reduce duplication of this code?

Reply:
:thumbsup: makes sense. Should I amend the commit, or add another?

Reply:
I think it is up to you. I prefer additional commit :)

Reply:
as requested